src/components/DiffView/index.spec.tsx

```diff
@@ -572,6 +618,44 @@ describe(__filename, () => {
     expect(getWidgetNodes(widgets).length).toEqual(0);
   });
 
+  it('enables syntax highlighting for diffs when possible', () => {
+    const _tokenize = jest.fn(tokenize);
+    const root = renderWithLinterProvider({
+      _diffCanBeHighlighted: jest.fn(() => true),
```

Comment:
that or `jest.fn().mockReturnValue(true)`

Reply:
It hurts my fingers to type all that 😆 jk